mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -470,6 +476,12 @@ void ReductionOp::print(OpAsmPrinter &p) {
   p << ", " << getVector();
   if (getAcc())
     p << ", " << getAcc();
+
+  if (getFastmathAttr() &&
```

Comment:
Should there be a roundtrip test to check if the additional flags print and parse as expected?

Reply:
done